examples/bot.py

```diff
@@ -25,18 +29,20 @@ async def on_ready():
     print(f"Our latency is {round(client.latency)} ms.")
 
 
+# We can either pass in the event name or make the function name be the event name.
 @interactions.listen("on_message_create")
-async def name_this_however_you_want(message: interactions.Message):
+async def name_this_however_you_want(message_create: interactions.events.MessageCreate):
     # Whenever we specify any other event type that isn't "READY," the function underneath
     # the decorator will most likely have an argument required. This argument is the data
     # that is being supplied back to us developers, which we call a data model.
 
     # In this example, we're listening to messages being created. This means we can expect
-    # a "message" argument to be passed to the function, which will be the data model of such.
+    # a "message_create" argument to be passed to the function, which will be the data model of such.
```

Comment:
Message_create is an event model, not the actual message data model

Reply:
I have changed the comment for this.